DESCRIPTION

```diff
@@ -41,13 +41,16 @@ Imports:
 Suggests: 
     covr,
     knitr,
+    lifecycle,
```

Comment:
I'm not sure usethis itself needs to add lifecycle to Suggests?

Reply:
Since we link to a documentation topic in the lifecycle package, not having it in Suggest causes a CMD check failure on Travis.

Reply:
oops actually we now only link to the website, so we can remove it!